.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -59,14 +59,24 @@ body:
     validations:
       required: true
 
+  - type: input
+    id: versions
+    attributes:
+      label: Affects Version(s)
+      description: Which versions of Kyuubi are affected by this bug?
+      placeholder: >
+        Latest/Stable/v1.1.0/From v1.0.0/All versions
```

Comment:
Seems it's a little bit hard code to list all version, is it possible to show the existed branch list automatically ?

Reply:
I initially considered using checkboxes but the list would become longer and longer as the number of versions increases. It might be better to let users describe this in plain text.

Reply:
can we use `e.g. master/1.3.0/1.2.0/...` in `placeholder` ?

Reply:
As you wish.